cypress/support/pages/management/pageManagement.js

```diff
@@ -48,30 +41,32 @@ class Management {
     cy.get(Management.#searchbar).type(keyword)
   }
 
-  clickEditStudentButton() {
-    cy.get(Management.#editStudentButton).eq(0).click()
+  clickEditStudentButton(email) {
+    cy.contains('td', email)
+      .siblings()
+      .find('a')
+      .should('have.attr', 'data-testid', 'edit_student_button')
+      .click()
   }
 
-  clickEditTeacherButton() {
-    cy.get(Management.#editTeacherButton).eq(0).click()
+  clickEditTeacherButton(email) {
+    cy.contains('td', email)
+      .siblings()
+      .find('a')
+      .should('have.attr', 'data-testid', 'edit_teacher_button')
+      .click()
   }
 
-  changeStudentUserInformation() {
+  changeUsername(firstname, surname) {
     cy.get(Management.#firstNameEditForm).clear()
-    cy.get(Management.#firstNameEditForm).type('Alex')
+    cy.get(Management.#firstNameEditForm).type(firstname)
     cy.get(Management.#lastNameEditForm).clear()
-    cy.get(Management.#lastNameEditForm).type('Abramovic')
-    cy.get(Management.#emailEditForm).clear()
-    cy.get(Management.#emailEditForm).type('alex.abramovic@example')
+    cy.get(Management.#lastNameEditForm).type(surname)
   }
 
-  changeTeacherUserInformation() {
-    cy.get(Management.#firstNameEditForm).clear()
-    cy.get(Management.#firstNameEditForm).type('Amber')
-    cy.get(Management.#lastNameEditForm).clear()
-    cy.get(Management.#lastNameEditForm).type('Adams Young')
+  changeEmail(new_email) {
     cy.get(Management.#emailEditForm).clear()
-    cy.get(Management.#emailEditForm).type('amber.adams-young@example')
+    cy.get(Management.#emailEditForm).type(new_email)
```

Comment:
may be variable name can be in camel case --> e.g. 'newEmail'

Reply:
done